lib/cli/commands/create.ts

```diff
@@ -1,31 +1,59 @@
+import readline from 'readline';
 import { Arguments } from 'yargs';
+import { CreateNodeRequest, CreateNodeResponse } from '../../proto/xudrpc_pb';
 import { callback, loadXudInitClient } from '../command';
-import { CreateNodeRequest } from '../../proto/xudrpc_pb';
-import readline from 'readline';
 
 export const command = 'create';
 
 export const describe = 'create an xud node';
 
 export const builder = {};
 
+const formatOutput = (response: CreateNodeResponse.AsObject) => {
+  if (response.seedMnemonicList.length > 0) {
+    const numberedMnemonic = response.seedMnemonicList.map((value, index) => {
+      return `${index >= 9 ? '' : ' '}${index + 1}. ${value.padEnd(10)}`;
+    });
+    console.log('----------------------BEGIN XUD SEED---------------------');
+    for (let n = 0; n < response.seedMnemonicList.length / 4; n += 1) {
```

Comment:
Lots of magic numbers here, but at the same time I don't see a better way of doing it. Perhaps we could assign `const WORDS_PER_ROW = 4`?

Reply:
Sure I can do WORDS_PER_ROW